### DISCUSSION TITLE
1wallet discussions

### DISCUSSION THREAD

**Discussion by polymorpher** [polymorpher/one-wallet#95] [Announcements]
## 👋 Welcome!

"Discussions" is a new feature on GitHub. It appears to be ideal place to share ideas, ask questions, and offer suggestions on product features. You can convert issues to discussions automatically.

**Comment by arkadihard** [comment]
Добавьте, пожалуйста, возможность выбора языка.
А именно требуется язык (Русский) - 
Я готов сделать перевод, если вы предоставите мне файл.
------(EN)------
Please add the ability to select a language.
Namely, the language is required (Russian) -
I am ready to translate if you provide me with the file.

**Reply by polymorpher** [reply]
Thanks but there no translation file yet. Language text is hard coded. Perhaps some time later

**Comment by arkadihard** [comment]
There is a problem with access to the wallet created through a laptop in the mobile version. The mobile version offers to create a wallet. But there are no buttons to specify an existing one for authorization and login.

**Reply by polymorpher** [reply]
Sorry, can't understand the first and third sentences

**Comment by arkadihard** [comment]
When you try to replenish the wallet with binance, the tokens are not credited, and after a few days they are returned to binance.

**Reply by polymorpher** [reply]
what kind of tokens? Can you show the transaction?
